runtime/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSubscribeFactory.java

```diff
@@ -706,14 +691,12 @@ final class KafkaMessagesProxy
 
         private KafkaMessagesProxy(
             long originId,
-            long routedId,
-            String16FW topic,
             MqttKafkaRouteConfig routeConfig,
             MqttSubscribeProxy mqtt)
         {
             this.originId = originId;
-            this.routedId = routedId;
-            this.topic = topic;
+            this.routedId = routeConfig.id;
+            this.topic = routeConfig.messages;
```

Comment:
Use `route` instead of `routeConfig` here, as it is easier to read and there is no ambiguity.